app/controllers/v1/sessions_controller.rb

```diff
@@ -150,6 +150,7 @@ def mhv_unverified_validation(user_session_form)
     end
 
     def render_login(type)
+      Rails.logger.info("Render Login Type: #{type}")
```

Comment:
We have a log that'll give you this information and more later in this function, this log isn't really giving you much...

`SSOe: SAML Request`